integration-tests/07_course.ts

```diff
@@ -390,3 +393,44 @@ void test('Add / Remove another instructor', async () => {
         subcourseDeleteInstructor(subcourseId: ${subcourseId} studentId: ${instructor2.student.id})
     }`);
 });
+
+void test('Find shared course', async () => {
+    const { courseId, client } = await subcourseOne;
+
+    await client.request(`
+        mutation MarkCourseShared {
+            courseMarkShared(courseId: ${courseId}, shared: true){id}
+        }
+    `);
+
+    const { templateCourses } = await adminClient.request(`
+        query FindSharedCourses {
+            templateCourses(search: "", take: 1) {
+              id
+            }
+        }
+    `);
+
+    assert.ok(templateCourses.some((it) => it.id === courseId));
+});
+
+void test('Test template course search', async () => {
+    const { instructor: instructor1 } = await screenedInstructorTwo;
+    const { instructor, courseId } = await subcourseOne;
+
+    await prisma.course.update({ where: { id: courseId }, data: { shared: false } });
```

Comment:
the two tests depend on each other - if they are run in parallel or in a different order, they both modify subcourseOne - either use one large test which is then sequential, or use two different subcourses

Reply:
Also please avoid updating the database in tests - we want to test the GraphQL interface so prefer that

Reply:
> the two tests depend on each other - if they are run in parallel or in a different order, they both modify subcourseOne - either use one large test which is then sequential, or use two different subcourses

Do multiple calls of subcourse one return the same subcourse object?

How can I obtain a second subcourse?